### PR TITLE
[11] Ci ign rename

### PR DESCRIPTION
**Pull request by nkoenig**
Signed-off-by: Nate Koenig <nate@openrobotics.org>

Rename `Ignition` to `Gazebo` in `ci.md` and `contributing.md`

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by chapulina — APPROVED** [review]

File: contributing.md [hunk 1]
```diff
@@ -545,14 +545,14 @@ You can find more details [here](https://github.com/colcon/colcon-sanitizer-repo
 
 #### Compiling the code and running the tests
 
-Create a Ignition workspace for ASAN/TSAN related tasks. Refer to the [offical documentation to get
-all the Ignition Robotics sources](https://gazebosim.org/docs/latest/install_ubuntu_src#getting-the-sources).
+Create a Gazebo workspace for ASAN/TSAN related tasks. Refer to the [offical documentation to get
+all the Gazebo Robotics sources](https://gazebosim.org/docs/latest/install_ubuntu_src#getting-the-sources).
```
Comment (methylDragon) [inline contributing.md:549]: Probably?

```suggestion
all the Gazebo sources](https://gazebosim.org/docs/latest/install_ubuntu_src#getting-the-sources).
```

**Review by methylDragon — CHANGES_REQUESTED** [review]

File: contributing.md (hunk 1 above)
Comment (nkoenig) [reply contributing.md:549]: We are not using `Robotics` as a modifier to `Gazebo`, right @chapulina?

Comment (chapulina) [reply contributing.md:549]: Yeah nice catch @methylDragon , we've been dropping the Robotics everywhere.

Comment (nkoenig) [reply contributing.md]: Thanks, removed.

**Pull request merged**